src/scripts/addserver.js

```diff
@@ -0,0 +1,65 @@
+define(["appSettings", "loading", "browser", "emby-linkbutton"], function(appSettings, loading, browser) {
+    "use strict";
+
+    function handleConnectionResult(page, result) {
+        loading.hide();
+        switch (result.State) {
+            case "SignedIn":
+                var apiClient = result.ApiClient;
+                Dashboard.onServerChanged(apiClient.getCurrentUserId(), apiClient.accessToken(), apiClient), Dashboard.navigate("home.html");
```

Comment:
Let's split in two lines
```suggestion
                Dashboard.onServerChanged(apiClient.getCurrentUserId(), apiClient.accessToken(), apiClient);
                Dashboard.navigate("home.html");
```